resident/resident-service/src/test/java/io/mosip/resident/test/validator/RequestValidatorTest.java

```diff
@@ -617,6 +618,7 @@ public void testGetStatusValidIndividualType() throws Exception {
 
 	}
 
+	@Ignore
```

Comment:
Why is this test case ignored ?

Reply:
The test is ignored because the test is expecting a rid validation will throw an exception. As of now, we have returned true always until we integrate new RID/App ID format validation.

Reply:
ok, from next time let's add these details to the commits and PR descriptions